mmv1/products/iam3/PrincipalAccessBoundaryPolicy.yaml

```
@@ -0,0 +1,157 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'PrincipalAccessBoundaryPolicy'
+description: Description
```

```suggestion
description: An IAM principal access boundary policy resource.
```

Copied from https://cloud.google.com/iam/docs/reference/rest/v3beta/organizations.locations.principalAccessBoundaryPolicies

---

Could you please also add these fields to the file, where the capitalised text needs to be replaced:

```
references:
  guides:
   # Link to quickstart in the API's Guides section. For example:
   # 'Create and connect to a database': 'https://cloud.google.com/alloydb/docs/quickstart/create-and-connect'
    'QUICKSTART_TITLE': 'QUICKSTART_URL'
  # Link to the REST API reference for the resource. For example,
  # https://cloud.google.com/alloydb/docs/reference/rest/v1/projects.locations.backups
  api: 'API_REFERENCE_URL'
```

They're described in [this section](https://googlecloudplatform.github.io/magic-modules/develop/resource/#add-a-resource) of the contribution guide and helps users find GCP documentation from the resource's provider documentation page.